core/walkmod.xml

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE walkmod PUBLIC "-//WALKMOD//DTD" "http://www.walkmod.com/dtd/walkmod-1.1.dtd">
+<walkmod>
```

Comment:
walkmod is like maven - it uses the execution directory to find the configuration. If it is a strict requirement, I can add an option for this.